IBPSA/Fluid/FixedResistances/CheckValve.mo

```diff
@@ -0,0 +1,96 @@
+within IBPSA.Fluid.FixedResistances;
+model CheckValve "Hydraulic one way valve"
+
+  extends IDEAS.Fluid.BaseClasses.PartialResistance(

```

Comment:
Please extend the IBPSA counterpart of this model, i.e. `IBPSA.Fluid.BaseClasses.PartialResistance`. Also replace all other references to IDEAS.